package.json

```diff
@@ -34,7 +36,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "server": "node public/getPosts.js"
```

Comment:
Please remove this. We will not need this command anymore.

Reply:
Done